### DISCUSSION TITLE
How to turn off ExplorerPatcher if Windows not loading?

### DISCUSSION THREAD

**Discussion by MaxGuH** [valinet/ExplorerPatcher#1829] [Q&A] [answered]
So. Yesterday I decided to update my Windows since all updates were put on a hold for a couple months. Today when I decided to fire up my pc, after suspiciously long user login screen is stuck in infinitely blinking main screen. Nothing loads after this. (see video below).
https://drive.google.com/file/d/11rfUkoG9sV3daJZz9vm9Lo5cc9Ks2n8L/view?usp=sharing
Nothing clickable at desktop. Luckily, ctrl+alt+del works so I can access Task Manager. So, main question, can I turn ExplorerPatcher off through Processes tab? Or Services tab? 
Any help will be much appreciated. Thank you in advance!

**Comment by MaxGuH** [comment]
P.S. It worth to mention that all my notifications were off, so there is a high chance ExplorerPatcher was not updated for a while. For a long while.

**Comment by MaxGuH** [comment, marked as answer]
Problem solved. If anyone interested, through Task Manager I did "Run new task" - "Browse" - located ep_setup.exe - letter by letter (with a blinking & constantly refreshing bar that was a pain, trust me) renamed it to ep_uninstall.exe - "OK".

**Comment by JsLth** [comment]
Another solution, if you're having trouble locating ep_setup.exe, is to run your browser from the task manager (through "run new task") and just download and install the latest EP update. Fixed all of my blinking issues a second into the installation process.
